subsys/fs/shell.c

```diff
@@ -299,23 +286,23 @@ static int cmd_read(const struct shell *shell, size_t argc, char **argv)
 			break;
 		}
 
-		shell_fprintf(shell, SHELL_NORMAL, "%08X  ", offset);
+		shell_print(shell, "%08X  ", offset);
 
 		for (i = 0; i < read; i++) {
-			shell_fprintf(shell, SHELL_NORMAL, "%02X ", buf[i]);
+			shell_print(shell, "%02X ", buf[i]);
 		}
 		for (; i < sizeof(buf); i++) {
-			shell_fprintf(shell, SHELL_NORMAL, "   ");
+			shell_print(shell, "   ");
 		}
 
-		shell_fprintf(shell, SHELL_NORMAL, " ");
+		shell_print(shell, " ");
 
```

Comment:
In order to avoid calling `shell_print` many times in the loop you can consider replacing:
```
for (; i < sizeof(buf); i++) {
			shell_fprintf(shell, SHELL_NORMAL, "   ");
			shell_print(shell, "   ");
		}
		shell_print(shell, " ");
```
with
```
i = sizeof(buf) - i;
shell_print(shell, "%*c ", i*3);
```

Reply:
This is not merge stop, just improvement proposal.